docs/sources/datasources/aws-cloudwatch/query-editor/index.md

```diff
@@ -92,6 +92,10 @@ For example, to apply arithmetic operations to a metric, apply a unique string i
 
 > **Note:** If you use the expression field to reference another query, like `queryA * 2`, you can't create an alert rule based on that query.
 
+##### Period macro
+
+If you're using a CloudWatch [`SEARCH`](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/search-expression-syntax.html) expression, you may want to use the `$__period_auto` macro rather than specifying a period explicitly. The `$__period_auto` macro will resolve to a [CloudWatch period](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricData.html) that is suitable for the choosen time range. It will use the same algorithm for calculating period as is used when the **Period** field is set to `auto`. Read more about that [here](#period).
```

Comment:
Sorry I'm not sure I follow this bit: `It will use the same algorithm for calculating period as is used when the **Period** field is set to "auto"`

What is the difference between the Period field described in the read more section and this one? This one is used in search expressions and the other one is used in other queries?

Reply:
I believe this relates to the way that CloudWatch calculates the period, when you don't specify it explicitly, or specify it as "auto".